projects/plugins/jetpack/changelog/update-remove-pay-with-paypal-widget-from-blocks

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: compat
+
+Pay with PayPal Widget - hide widget from Legacy Widget block
```

Comment:
This is a tiny thing.

I think we generally try to use `:` instead of `-` in the Jetpack changelog file.

```suggestion
Pay with PayPal Widget: hide widget from Legacy Widget block
```